kaznet/apps/ona/api.py

```diff
@@ -751,18 +751,21 @@ def sync_deleted_projects(usernames: list):
     If it finds any, it deletes them locally
     """
     onadata_project_pks = []
+
     for username in usernames:
         onadata_projects = get_projects(username=username)
+
         if isinstance(onadata_projects, list) and onadata_projects:
```

Comment:
@DavisRayM it looks like if `onadata_projects` is `None` here then we should abort.